src/http_client_casablanca.cpp

```diff
@@ -135,7 +135,7 @@ class http_client::impl
         #else
             #define USER_AGENT_PLATFORM
         #endif
-        m_userAgent = L"Poedit/" make_wide_str(POEDIT_VERSION) USER_AGENT_PLATFORM;
```

Comment:
Why?

Reply:
Needed by Crowdin for analytics

Reply:
Well, now I fully understand your concern (sorry for being slow in this). Then maybe I just chosen wrong place of changing it, so  how about to revert above back to your original while adding `userAgent = ""` argument to `http_client` [public constructor here](https://github.com/vslavik/poedit/blob/495a3969a063649eb2466cdfa27f00954fca0929/src/http_client.h#L124)?

Reply:
I really don't see a reason to change the User-Agent. If some server wishes to analyze Poedit's usage, it can just as easily grep for `Poedit/2` as it can for `poedit-crowdin` or whatever...

Reply:
Reverted back to "Poedit/..." a9a9dda400cbcfcf2d6863d10d64410dc725ae71